composer.json

```diff
@@ -17,7 +17,6 @@
         "nextcloud/coding-standard": "^1",
         "psalm/phar": "^4.3",
         "nextcloud/ocp": "dev-master",
-        "guzzlehttp/guzzle": "^7.5.1",
```

Comment:
Mmh, this is require-dev, though. It shouldn't be shipped, no?

Reply:
ok, I see kiota requires guzzle

Still, can you please keep in the require-dev for guzzle, so our psalm tests work?